contracts/core/Allo.sol

```diff
@@ -595,8 +595,6 @@ contract Allo is IAllo, Native, Initializable, Ownable, AccessControlUpgradeable
         Pool storage pool = pools[_poolId];
         address _token = pool.token;
 
-        if (_token == NATIVE && msg.value < _amount) revert ETH_MISMATCH();
```

Comment:
wouldn't it make sense to more senst keep this check on the internal function and instead remove the check on the external function `fundPool` ?

Reply:
In `_createPool` seems like this was only checked if baseFee > 0 so I ended up keeping the check in `_fundPool` and only removed it from `fundPool`